packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/pages/api/editing/config.ts

```diff
@@ -10,6 +10,7 @@ import metadata from 'temp/metadata.json';
 const handler = new EditingConfigMiddleware({
   components,
   metadata,
+  enableEditingMetadata: true,
```

Comment:
1. Please add an upgrade guide entry for customers to apply this param
2. Please consider renaming this param to be more descriptive for customers. Maybe it can be an "EditMode" type parameter called "pagesEditMode:" and have values Chromes/Metadata, with a short comment saying "Metadata" is the newer mode. This way devs will have more context if they wish to make the switch here

Reply:
I think that this name also looks good, but you don't need to add it in a API handler, since it's enabled by default in a service.
You can just remove this property, but customer will need to set "enableEditingMetadata: false" if they would like to disable a new integration

Reply:
Good idea, we can add _editMode_ property, and by default it will be _metadata_ in a service. We don't need to pass any new options in the app, by default